docs/variables.md

```diff
@@ -32,6 +32,11 @@ This page gives an overview of all possible variables that can be put in a `terr
 | [default_worker_instance_type](#default_worker_instance_type)                               | Nodes                | No  | "" |
 | [group_enabled](#group_enabled)                                                             | Nodes                | No  | false |
 | [spot_nodes_enabled](#spot_nodes_enabled)                                                   | Nodes                | No  | false |
+| [min_nodes](#min_nodes)                                                                     | Nodes                | No  | 0 |
+| [desired_nodes](#desired_nodes)                                                             | Nodes                | No  | 0 |
```

Comment:
value for desires_nodes been shared for both nodes and spot nodes ASG configuration. can this be separated out into two variables?

Reply:
This is an issue, but It should probably be addressed in a separate change